tests/Integration/PrestaShopBundle/Translation/Provider/ExternalModuleLegacySystemProviderTest.php

```diff
@@ -37,7 +37,7 @@
  */
 class ExternalModuleLegacySystemProviderTest extends KernelTestCase
 {
-    const MODULE_NAME = 'translation_test';
+    const MODULE_NAME = 'translationtest';
```

Comment:
underscore are valid in module name: why do we need to change that ?

Reply:
https://github.com/PrestaShop/PrestaShop/pull/14845

As stated in the PR's description, you should really read @eternoendless 's PR

Reply:
Underscores are *NOT* valid and *WON'T* work.

Reply:
I mean, you can have underscores in your module name but you won't be able to translate it. Here's why:

Given a module `my_module`, its domain name would have to be either `MyModule` or `Mymodule`, because underscores are not allowed (at least not in that part). 

`MyModule` wouldn't be valid because internally domains are stored without dots, meaning that `MyModule` would be broken down into a subdomain like `My.Module`. `Mymodule` won't work either because we have no way of transforming `Mymodule` back into `my_module` to find the module directory and read its translation files.